android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java

```diff
@@ -1031,6 +1031,7 @@ public void initialize(NewTabPageManager manager, Activity activity,
         assert (activity instanceof BraveActivity);
         mActivity = activity;
         ((BraveActivity) mActivity).dismissShieldsTooltip();
+        ((BraveActivity) mActivity).setNewTabPageManager(manager);
```

Comment:
@tapanmodh how would it work for existing users? existing user may have website loaded when they open the app after upgrade.

Reply:
when user open the app on NTP and app count is >=7 then it will display the dialog. wdyt?